ArduPlane/quadplane.cpp

```diff
@@ -691,8 +688,7 @@ bool QuadPlane::setup(void)
         // this is a copter tailsitter with motor layout specified by frame_class and frame_type
         // tilting motors are not supported (tiltrotor control variables are ignored)
         if (tilt.tilt_mask != 0) {
-            hal.console->printf("Warning tilting motors not supported, setting tilt_mask to zero\n");
-            tilt.tilt_mask.set(0);
```

Comment:
This has just been lost - is that good?

Reply:
It just emits a warning now, instead of forcing a param to zero.
Since copter tailsitters aren't supposed to have tilting motors, it shouldn't matter.